.editorconfig

```diff
@@ -0,0 +1,42 @@
+# https://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = tab
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
+
+[**/*.yml]
+indent_style = space
+indent_size = 2
+
+[**/*.md]
+trim_trailing_whitespace = false
+indent_style = space
+indent_size = 2
+
+[**/*.json]
+indent_style = space
+indent_size = 2
+
+[**/*.php]
+indent_style = space
+indent_size = 4
+
+[**/*.neon]
+indent_style = space
+indent_size = 4
+
+[**/*.js]
+indent_style = space
+indent_size = 2
+
+[composer.lock]
+indent_style = unset
```

Comment:
Please consider using only spaces (or only tabs).

Reply:
I like to have the config files (.json, .yaml...) more compact, so I'll keep them with 2 spaces only. Markdown as well. With anything else I prefer spaces over tabs.